docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -27,7 +27,7 @@ That index must be deleted separately.
 
 [float]
 [[dfa-update-limitations]]
-=== Cannot update a {dfanalytics-job}
+=== A {dfanalytics-job} configuration cannot be updated
```

Comment:
Is there some part of the job that *can* be updated? If not, I suggest simplifying to something like this:
```suggestion
=== {dfanalytics-jobs-cap} cannot be updated
```